contracts/KashiPair.sol

```diff
@@ -394,7 +394,7 @@ contract KashiPair is ERC20, BoringOwnable, IMasterContract {
 
         (totalBorrow, part) = totalBorrow.add(amount.add(feeAmount), true);
         userBorrowPart[msg.sender] = userBorrowPart[msg.sender].add(part);
-        emit LogBorrow(msg.sender, to, amount.add(feeAmount), part);
+        emit LogBorrow(msg.sender, to, amount, feeAmount, part);
```

Comment:
This makes accounting more straightforward. Otherwise I may be confused with a higher `amount` value.`feeAmount` makes this clear. Also this helps a lot for accounting in the state machine without doing nasty hacks.